src/file-uploader/file-uploader.component.ts

```diff
@@ -231,6 +231,9 @@ export class FileUploader {
 
 		if (!this.accept.length) {
 			transferredFiles.forEach(file => {
+				if ((this.files.size >= 1) && !this.multiple) {
```

Comment:
Perhaps we could set up the `transferredFiles` correctly for this case, instead of bailing out in the `forEach` ... That way we only have on conditional to track, instead of two 🤔

Reply:
good point!